cmd/clusterctl/api/v1alpha3/metadata_type.go

```diff
@@ -32,29 +32,29 @@ type Metadata struct {
 	ReleaseSeries []ReleaseSeries `json:"releaseSeries"`
 }
 
-// ReleaseSeries maps a provider release series (major/minor) with a ClusterAPIVersion
+// ReleaseSeries maps a provider release series (major/minor) with a API Version of Cluster API (contract).
 type ReleaseSeries struct {
 	// Major version of the release series
 	Major uint `json:"major,omitempty"`
 
 	// Minor version of the release series
 	Minor uint `json:"minor,omitempty"`
 
-	// ClusterAPIVersion indicates the cluster API supported version.
-	ClusterAPIVersion string `json:"clusterAPIVersion,omitempty"`
+	// Contract defines the API Version of Cluster API (contract) supported by the ReleaseSeries.
+	Contract string `json:"contract,omitempty"`
```

Comment:
Renaming `ClusterAPIVersion`  into `Contract` so we are consistent with the term used in https://github.com/kubernetes-sigs/cluster-api/pull/2267 / less confusing

Reply:
Just out of my own curiosity, when you reference `Cluster API (contract)` are you referring to the clusterctl provider contract?

Reply:
I'm assuming there will be some documentation explaining `Cluster API (contract)` in the clusterctl upgrade section of the book.

Reply:
The API Version of Cluster API (contract) is v1alpha2, v1alpha3 etc. If this is not clear enough in the doc we should iterate and make it crystal clear